src/redisearch_rs/inverted_index/src/lib.rs

```diff
@@ -1791,3 +1791,25 @@ impl<'index, IR: NumericReader<'index>> NumericReader<'index> for FilterGeoReade
 
 #[cfg(test)]
 mod tests;
+
+/// Field expiration predicate used when checking fields.
```

Comment:
@chesedo : I'm not fan of keep adding those to the `inverted_index` crate. 
The `rqe_iterator` one would probably be a better fit but that would introduce a circular dependency in `types_ffi` (used to generate the C bindings) as `rqe_iterator` depends on `types_ffi`.

What do you think? Should we have `types_rs` crates for this kind of types?

Reply:
We should group by function, not by "kind".
This probably belongs to a `field` crate, that's later going to be the home of `FieldSpec` and a bunch of other things.